app/src/Application/DbSettingsObj.cpp

```diff
@@ -0,0 +1,74 @@
+#include "DbSettingsObj.h"
+
+using namespace Status::Application;
+
+DbSettingsObj::DbSettingsObj(StatusGo::Settings::SettingsDto rawData)
+    : QObject(nullptr)
+    , m_data(std::move(rawData))
+{
+}
+
+QString DbSettingsObj::address() const
+{
+    return m_data.address;
+}
+
+void DbSettingsObj::setAddress(const QString& value)
```

Comment:
Few years ago i've discovered [this]( https://gist.github.com/vlansco/0b4cd5f765121355e3413132ee3a6259) macro and it can be used with Q_PROPERTY to generate setters and getters. And there is also possibility to implement it using [templates](https://github.com/Midgrad/kjarni/blob/682eea6b9378d685958b418506efb5f70a16d95e/include/base/utils/magic_property.hpp).

Reply:
@MishkaRogachev I was thinking about adding that now, but will create a new PR for that. Plan to introduce more convenient macros than those. Macros where you will, beside adding getter, signal and setter, decide in a single line whether to use the signal created based on prop name, with or without an argument or share the same signal among multiple properties which is also very often a need.